### PR TITLE
Implement Login

### PR DESCRIPTION
**Pull request by joshtrigger**
- configure routing for auth module
 - create login component

### CODE REVIEW

File: src/app/auth/services/auth.service.spec.ts [hunk 1]
```diff
@@ -56,6 +56,29 @@ describe("AuthService", () => {
     expect(req.request.method).toBe("GET");
   });
 
+  it("should should return true when no user is logged in", () => {
```
Comment (Bbaale15) [inline src/app/auth/services/auth.service.spec.ts]: This test description is a little confusing to me, "should return true when no user is logged in" shouldn't it return false instead? Also, remove the second should.

**Review by Bbaale15 — CHANGES_REQUESTED** [review]
Apart from the comment, I added on your unit test other things look good.

File: src/app/auth/services/auth.service.spec.ts (hunk 1 above)
Comment (joshtrigger) [reply src/app/auth/services/auth.service.spec.ts]: thank you, I'm not it

**Comment by joshtrigger** [comment]
Hello @Bbaale15 I have implemented the feedback.

**Pull request merged**